src/Domain/User/User.ts

```diff
@@ -33,6 +34,21 @@ export class User {
   })
   pwNonce: string
 
+  @Column({
+    name: 'encrypted_server_key',
+    length: 255,
+    type: 'varchar',
+    nullable: true
+  })
+  encryptedServerKey: string | null
+
+  @Column({
+    name: 'encryption_version',
```

Comment:
Should be called `server_encryption_version` as to not be conflated with `user.version` which relays client encryption version.